src/sentry/static/sentry/app/components/dateTime.jsx

```diff
@@ -30,12 +31,12 @@ class DateTime extends React.Component {
     }
 
     if (clock24Hours) {
-      return 'MMMM D YYYY HH:mm:ss z';
+      return `MMMM D YYYY HH:mm:ss${!utc ? ' z' : ''}`;
     }
 
     // Oct 26, 2017 11:30:30 AM
     if (seconds) {
-      return 'll LTS z';
+      return `ll LTS${!utc ? ' z' : ''}`;
```

Comment:
We could leave the timezone in, it would display `UTC`... thoughts?

Reply:
It would be clearer which timezone the customer was looking at. Could be confusing to see times up to 10hours in the past/future.